### PR TITLE
Improve fines for ui

### PR DESCRIPTION
**Pull request by Jonybang**

### CODE REVIEW

File: contracts/FundStorage.sol [hunk 1]
```diff
@@ -241,32 +275,48 @@ contract FundStorage is Permissionable {
...
-      string memory description
-    )
+  external
+  view
+  returns (
+    bytes32 contractType,
+    bytes32 abiIpfsHash,
+    string memory description
+  )
   {
```
Comment (chebykin) [inline contracts/FundStorage.sol]: the padding is ok

File: contracts/FundStorage.sol [hunk 2]
```diff
@@ -68,7 +70,13 @@ contract FundStorage is Permissionable {
...
     uint256 total;
     // Assume ETH is address(0x1)
-    mapping(address => uint256) tokenFines;
+    mapping(address => MemberFineItem) tokenFines;
+  }
+
+  struct MemberFineItem {
+    uint256 amount;
+    uint256[] proposals;
+    address[] proposalsManagers;
```
Comment (chebykin) [inline contracts/FundStorage.sol]: Here we have config contract. It's a really bad idea to cache values for UI here. This information could be kept in Proposal manager contract or any other contract.

File: contracts/FundStorage.sol (hunk 1 above)
Comment (Jonybang) [reply contracts/FundStorage.sol]: fixed

File: contracts/FundStorage.sol (hunk 2 above)
Comment (Jonybang) [reply contracts/FundStorage.sol]: todo added

**Pull request merged**